src/main/resources/application.yaml

```diff
@@ -1,10 +1,12 @@
 spring:
+  main:
+    allow-bean-definition-overriding: true
   datasource:
-    url: 'jdbc:h2:file:./target/database/h2'
+    url: 'jdbc:postgresql://localhost:5432/hapi_fhir'
     #url: jdbc:h2:mem:test_mem
-    username: sa
-    password: null
-    driverClassName: org.h2.Driver
+    username: postgres
+    password: root
+    driverClassName: org.postgresql.Driver
```

Comment:
The secrets in this should be moved to a private repository or secrets file. @dubdabasoduba will liase with SRE and decided how to move forward with this

Reply:
@manutarus @bennsimon Do you have an idea of how to use secret files to provide these creds?